### DISCUSSION TITLE
Is having our twitch cookies PKL file online (where anyone can access it) safe?

### DISCUSSION THREAD

**Discussion by ArrickGit** [Tkd-Alex/Twitch-Channel-Points-Miner-v2#527] [Q&A] [answered]
Hello, I am relatively new to this and was planning to use replit to consistently run the script on the cloud. However, this will result in my cookies PKL file being expose online as you are required to pay in order to private your files. So I was wondering if it is safe to have my cookies PKL file saved online where anyone is able access and download it?
So sorry if this is an inappropriate question >.<
Thank you for your time and appreciate any replies!

**Comment by ColinShark** [comment, marked as answer]
No, anyone who can access your cookie file can authenticate as you, doing everything you can. If you have payment information on twitch, a malicious actor could subscribe you to dozens of random streamers, emptying your wallet.

You either want to pay for repl.it, or use another hoster where files aren't public.

From what I've heard heroku should support this program. If you're well versed enough with the command line, I can also recommend [Uberspace](https://uberspace.de/en).

**Reply by ArrickGit** [reply]
Oh alright thank you! I'll try out uberspace

**Reply by ArrickGit** [reply]
Also, does changing my twitch password help to reset my cookies?

**Reply by ColinShark** [reply]
Resetting your passwords logs all sessions out and will require re-authentication. Sadly Twitch doesn't offer a button to log all other sessions out without changing your password.
